services/scenario-manager/entities/entities.go

```diff
@@ -66,20 +73,21 @@ type Service struct {
 
 // Topology Configuration
 type TopologyConfig struct {
-	Id               string    `json:"id"`
-	Name             string    `json:"name"`
-	TopoType         string    `json:"type"`
-	NumberOfVhosts   uint      `json:"number_of_vhosts"`
-	NumberOfRacks    uint      `json:"number_of_racks"`
-	VhostsPerRack    uint      `json:"vhosts_per_rack"`
-	DataPlaneCidr    string    `json:"data_plane_cidr"`
-	NumberOfGateways uint      `json:"number_of_control_plane_gateways"`
-	GatewayIPs       []string  `json:"control_plane_gateway_ips"`
-	Images           []Image   `json:"images"`
-	VNodes           []VNode   `json:"vnodes"`
-	VLinks           []VLink   `json:"vlinks"`
-	CreatedAt        time.Time `json:"created_at"`
-	UpdatedAt        time.Time `json:"updated_at"`
+	Id               string        `json:"id" swaggerignore:"true"`
```

Comment:
What 's the difference for swaggerignore between here and status in the later parts?